src/Stryker.Core/Stryker.Core.UnitTest/stryker-config.json

```diff
@@ -5,9 +5,9 @@
       "!Logging/**",
       "!**/FSharp*.*"
     ],
-    "dashboard-project": "github.com/stryker-mutator/stryker-net",
-    "dashboard-version": "master",
-    "dashboard-fallback-version": "master",
-    "dashboard-module": "core"
+    "project-info": {
+      "name": "github.com/stryker-mutator/stryker-net",
```

Comment:
This is not needed, we automatically determine the project name from the project using SourceLink.

This is set using Reproducible Builds here: https://github.com/stryker-mutator/stryker-net/blob/master/src/Directory.Build.props#L15